docs/en/dgs/dataloader.md

```diff
@@ -1,3 +1,158 @@
-# Load data from source
+# Loading From Your Data Source
 
-(comming soon...)
+By using kafka queues as the entry for streaming graph updates, the Dynamic-Graph-Service can decouple the different
+data sources and sampling workers.
+you should develop your own dataloader to read you data source and produce the graph updates into output kafka queues
+for further consuming.
+
+## Rules to Follow
+
+When you process you source data and produce them in your dataloader, you should follow these rules:
```

Comment:
When you process you source data and produce them in your dataloader, you should follow these rules: --> When processing source data in the dataloader, these rules should be followed:

Reply:
done